src/stability_sdk/animation.py

```diff
@@ -270,25 +271,26 @@ def interpolate_frames(
     ratios = np.linspace(0, 1, interp_factor+1)[1:-1].tolist()
     for i in range(len(frame_files) - 1):
         shutil.copy(frame_files[i], os.path.join(out_path, f"frame_{i * interp_factor:05d}.png"))
-        frame1 = cv2.imread(frame_files[i])
-        frame2 = cv2.imread(frame_files[i + 1])
-        yield cv2_to_pil(frame1)
+        frame1 = Image.open(frame_files[i])
+        frame2 = Image.open(frame_files[i + 1])
+        yield frame1
         tweens = context.interpolate([frame1, frame2], ratios, interp_mode)
         for ti, tween in enumerate(tweens):
-            cv2.imwrite(os.path.join(out_path, f"frame_{i * interp_factor + ti + 1:05d}.png"), tween)
-            yield cv2_to_pil(tween)
+            tween.save(os.path.join(out_path, f"frame_{i * interp_factor + ti + 1:05d}.png"))
+            yield tween
 
     # copy final frame
     shutil.copy(frame_files[-1], os.path.join(out_path, f"frame_{(len(frame_files)-1) * interp_factor:05d}.png"))        
 
-def mask_erode_blur(mask: np.ndarray, mask_erode: int, mask_blur: int) -> np.ndarray:
+def mask_erode_blur(mask: Image.Image, mask_erode: int, mask_blur: int) -> Image.Image:
+    mask = np.array(mask)
     if mask_erode > 0:
         ks = mask_erode*2 + 1
         mask = cv2.erode(mask, cv2.getStructuringElement(cv2.MORPH_ELLIPSE, (ks, ks)), iterations=1)
     if mask_blur > 0:
         ks = mask_blur*2 + 1
         mask = cv2.GaussianBlur(mask, (ks, ks), 0)
```

Comment:
It's to be replaced with PIL.ImageFilter.GaussianBlur() and PIL.ImageFilter.MinFilter(), right?